### PR TITLE
TST:  e2e: watch-only -> receive (& receive with amount)

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/24cgbg3thgy6w5jczkhpmhega8

[download apk](http://filestorage.bluewallet.io:1488/TSTwatchonly-1611449610.apk)

File: tests/e2e/bluewallet.spec.js [hunk 1]
```diff
@@ -547,6 +547,36 @@ describe('BlueWallet UI Tests', () => {
     process.env.TRAVIS && require('fs').writeFileSync(lockFile, '1');
   });
 
+  it('can import zpub as watch-only, go to transactions list, go into Receive modal, and set custom amount. Dismiss modal and go to WalletsList.', async () => {
```
Comment (Overtorment) [inline tests/e2e/bluewallet.spec.js:550]: can it be a part of other testcase? 
every testcase and wallet import takes a lot of time on CI

File: tests/e2e/bluewallet.spec.js [hunk 2]
```diff
@@ -547,6 +547,36 @@ describe('BlueWallet UI Tests', () => {
...
+    await yo('BitcoinAddressQRCodeContainer');
+    await yo('BlueCopyTextToClipboard');
+    await element(by.id('SetCustomAmountButton')).tap();
+    await element(by.id('BitcoinAmountInput')).typeText('1');
+    await element(by.id('CustomAmountDescription')).typeText('test');
+    await element(by.id('CustomAmountSaveButton')).tap();
+    await sup('1 BTC');
+    await sup('test');
+    await yo('BitcoinAddressQRCodeContainer');
+    await yo('BlueCopyTextToClipboard');
```
Comment (Overtorment) [inline tests/e2e/bluewallet.spec.js:573]: get the resulting bip21 string and verify it as well

File: tests/e2e/bluewallet.spec.js [hunk 3]
```diff
@@ -547,6 +547,36 @@ describe('BlueWallet UI Tests', () => {
...
+    await device.pressBack();
+    await device.pressBack();
```
Comment (Overtorment) [inline tests/e2e/bluewallet.spec.js:575]: this is only needed if after it comes another test in the same testcase

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/0dwawk0ue3k17a88rwd2ypae94

[download apk](http://filestorage.bluewallet.io:1488/TSTwatchonly-1611814098.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tda8f3ctyg4c14n13zv019qn78

[download apk](http://filestorage.bluewallet.io:1488/TSTwatchonly-1611824861.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1188zn16jefdc9wqb243netbmw

[download apk](http://filestorage.bluewallet.io:1488/TSTwatchonly-1611828240.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7ed6f70c2yjr5mtceucfu4cayg

[download apk](http://filestorage.bluewallet.io:1488/TSTwatchonly-1611864453.apk)

File: tests/e2e/bluewallet.spec.js [hunk 4]
```diff
@@ -512,6 +512,26 @@ describe('BlueWallet UI Tests', () => {
...
+    await element(by.id('SetCustomAmountButton')).tap();
+    await element(by.id('BitcoinAmountInput')).typeText('1');
+    await element(by.id('CustomAmountDescription')).typeText('Test');
+    await element(by.id('CustomAmountSaveButton')).tap();
+    await expect(element(by.text('1 BTC'))).toBeVisible();
+    await expect(element(by.text('Test'))).toBeVisible();
+    await expect(element(by.id('BitcoinAddressQRCodeContainer'))).toBeVisible();
+
+
+    await expect(element(by.text('bitcoin:bc1qtc9zquvq7lgq87kzsgltvv4etwm9uxphfkvkay?amount=1&label=Test'))).toBeVisible();
```
Comment (Overtorment) [inline tests/e2e/bluewallet.spec.js:532]: gj

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**